slack-base/src/main/java/com/hubspot/slack/client/models/response/migration/MigrationExchangeResponseIF.java

```diff
@@ -0,0 +1,25 @@
+package com.hubspot.slack.client.models.response.migration;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+import org.immutables.value.Value.Immutable;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+import com.hubspot.slack.client.models.response.SlackResponse;
+
+@Immutable
+@HubSpotStyle
+@JsonNaming(SnakeCaseStrategy.class)
+public interface MigrationExchangeResponseIF extends SlackResponse {
+  String getTeamId();
+
+  String getEnterpriseId();
+
+  Map<String, String> getUserIdMap();
+
+  Optional<Set<String>> getInvalidUserIds();
```

Comment:
One more thing. I wonder if it's easier to have `Set<String>` instead of `Optional` here. `Set<String> getInvalidUserIds` will be parsed as an empty set if no property is present in JSON at all. So we still can work with the entity but IMHO, it's a bit easier to work with a set that can be empty than with `Optional`. What do you think?

Reply:
Yes that will be better. I just didn't now that json can deserialize without exception. when set property is missing. I'll update it.

Reply:
Updated.